Jenkinsfile

```diff
@@ -66,6 +66,7 @@ node ('master') {
             // Build Raft
             stage("Build Raft") {
               sh "docker-compose up --build --abort-on-container-exit --force-recreate --renew-anon-volumes --exit-code-from raft-engine"
+              sh "docker-compose -f docker-compose-installed.yaml build"
```

Comment:
We should use the installed image to run tests, so we know we are releasing working artifacts (meaning you should delete the old `docker-compose up --build`.

Reply:
It seems that we have to run them in the first container as that's the only one that has cargo installed.

Reply:
You are right, you need to run the unit tests in the local image because you need `cargo`. You should be able to run the integration tests from the installed images though.